config/_default/params.json

```diff
@@ -20,7 +20,8 @@
   "footer-promo-mobile-height": 250,
   "footer-promo-mobile-images": [],
   "footer-promo-mobile-width": 500,
-  "newsletter-active": true,
+  "newsletter-active": false,
+  "takeover-active": true,
```

Comment:
I’m going to have to add a slot to Almanack to keep this from being overwritten. Seems like a good idea though so Chris can toggle it.